docs/docs/monitoring/analytics/getting-started-with-analytics.mdx

```diff
@@ -0,0 +1,154 @@
+---
+id: getting-started-with-analytics
+sidebar_label: Getting started
+title: Getting started with Analytics
+description:
+abstract: Rasa Pro Analytics is a data pipeline that extracts model, training,
+  and conversation data from the Rasa platform and streams the data to
+  your data warehouse for analysis.
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import RasaProLabel from "@theme/RasaProLabel";
+import RasaProBanner from "@theme/RasaProBanner";
+
+<RasaProLabel />
+
+<RasaProBanner />
+
+Analytics helps visualize and process Rasa assistant metrics in the
+tooling (BI tools, data warehouses) of your choice. Visualizations
+and analysis of the production assistant and its conversations allow
+you to assess ROI and improve the performance of the assistant over time.
+
+<div align="center">
+  <img
+    alt="Rasa Pro Analytics example dashboard"
+    src={useBaseUrl("/img/analytics/analytics-examples.png")}
```

Comment:
Is anybody working on an updated version of this image?

Reply:
i am not